app/src/js/components/ReingestGranules/BatchReingestCompleteContent.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const maxDisplayed = 10;
+
+const BatchReingestCompleteContent = ({
+  results,
+  error
+}) => {
+  const confirmation = `successfully reingested ${results.length > 1 ? 'these' : 'this'} granule${results.length > 1 ? 's' : ''}`;
```

Comment:
Not necessary, but perhaps we should should just place this directly in the markup so we know results is defined when checking for length

Reply:
yeah, good point.  But what if it's was function instead. to keep the markup cleaner?  feelings?

Reply:
Sure!

Reply:
e53a3b4a